testdata/generate.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+test -e
+# An example how-to re-generate testing certificates (because usually
+# TLS certificates have expiration dates and some day they will expire).
+#
+# The instruction is valid for:
+#
+# $ openssl version
+# OpenSSL 3.0.2 15 Mar 2022 (Library: OpenSSL 3.0.2 15 Mar 2022)
+
+cat <<EOF > domains.ext
+authorityKeyIdentifier=keyid,issuer
+basicConstraints=CA:FALSE
+keyUsage = digitalSignature, nonRepudiation, keyEncipherment, dataEncipherment
+subjectAltName = @alt_names
+[alt_names]
+DNS.1 = localhost
+IP.1 = 127.0.0.1
+EOF
+
+openssl req -x509 -nodes -new -sha256 -days 1024 -newkey rsa:2048 -keyout ca.key -out ca.pem -subj "/C=US/CN=Example-Root-CA"
+openssl x509 -outform pem -in ca.pem -out ca.crt
+
+openssl req -new -nodes -newkey rsa:2048 -keyout localhost.key -out localhost.csr -subj "/C=US/ST=YourState/L=YourCity/O=Example-Certificates/CN=localhost"
+openssl x509 -req -sha256 -days 1024 -in localhost.csr -CA ca.pem -CAkey ca.key -CAcreateserial -extfile domains.ext -out localhost.crt
```

Comment:
If possible, I would generate testing certificates without an expiration date.

Reply:
I can't find a way to do it. I've Increased the number of days to 8192. That should be enough:
```bash
$ openssl x509 -enddate -noout -in ca.pem 
notAfter=Oct 29 06:17:40 2044 GMT
$ openssl x509 -enddate -noout -in localhost.crt 
notAfter=Oct 29 06:17:40 2044 GMT
```

Reply:
Okay, thanks!